developer/src/kmc-copy/src/copier-messages.ts

```diff
@@ -19,9 +19,10 @@ const SevFatal = CompilerErrorSeverity.Fatal | Namespace;
  */
 export class CopierMessages {
   static FATAL_UnexpectedException = SevFatal | 0x0001;
-  static Fatal_UnexpectedException = (o:{e: any}) => m(
+  static Fatal_UnexpectedException = (o:{e: any}) => CompilerMessageSpecWithException(
```

Comment:
This was incorrect -- was giving a null error message, picked up by the new test :grin: